fabischapeli/fast-food-fast-v1
# #160232013 fetches a specific meal from the application

What does this PR do?

It fetches one specific meal from the app

Description of Task to be completed?

Should return one meal from the application and display it is json format

How should this be manually tested?

Fork this branch to your github, clone it to your machine.
Go to google chrome, more tools, extensions and add postman to google chrome.
On post man, make sure your server is running and add the method and the route you want to test.
It will return the information you need  else there is a problem in your code, or the server is not running

Any background context you want to provide?

The server is set  up using Flask mini framework

What are the relevant pivotal tracker stories?

#160232013
